src/circuit/gadget/ecc/chip.rs

```diff
@@ -0,0 +1,386 @@
+use super::EccInstructions;
+use crate::constants::{self, OrchardFixedBasesFull, ValueCommitV};
+use arrayvec::ArrayVec;
+use ff::Field;
+use halo2::{
+    arithmetic::CurveAffine,
+    circuit::{Cell, Chip, Layouter},
+    plonk::{Advice, Column, ConstraintSystem, Error, Fixed, Permutation, Selector},
```

Comment:
```suggestion
    plonk::{Advice, Any, Column, ConstraintSystem, Error, Fixed, Permutation, Selector},
```
(used by a suggestion below).